cmd/sand-agent/sand-agent.go

```diff
@@ -239,8 +239,18 @@ func ensureNetworks(ctx context.Context, c *config.Config, repo network.Reposito
 			MoveVeth:     true,
 		})
 		if err != nil {
-			log.WithError(err).Error("fail to ensure endpoint")
-			continue
+			// if we can't activate the endpoint because the netns path doesn't exist anymore, we
+			// just deactivate it. Otherwise we raise an error.
+			if strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
Not the way to check:

https://godoc.org/github.com/pkg/errors#Cause
https://godoc.org/os#IsNotExist

Reply:
Thanks I didn't found it when I checked :ok_hand: